spring-cloud-task-docs/src/main/asciidoc/batch.adoc

```diff
@@ -107,3 +107,14 @@ public DeployerStepExecutionHandler stepExecutionHandler(JobExplorer jobExplorer
 Spring Cloud Task provides the ability for batch jobs to emit informational messages.  This
 is covered in detail in the section
 <<stream.adoc#stream-integration-batch-events, Spring Batch Events>>.
+
+[[batch-failures-and-tasks]]
+== Batch Job Exit Codes
+
+As discussed before Spring Cloud Task applications support the ability to record the exit code of a task execution.
+However in cases where a user is running a Spring Batch Job within a task, regardless of how the Batch Job
+Execution completes the result of the task will always be zero when using a CommandLineRunner for example.  Keep in mind
```

Comment:
It won't be zero when using "a CommandLineRunner"...it will be zero when using the default Batch and Boot functionality.  I can change this on merge.